db/migrate/20240108202126_add_confirm_terms_to_site_settings.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+class AddConfirmTermsToSiteSettings < ActiveRecord::Migration[7.1]
+  def change
+    Setting.create!(name: 'ConfirmTerms') unless Setting.exists?(name: 'ConfirmTerms')
+
+    return if SiteSetting.exists?(setting: Setting.find_by(name: 'ConfirmTerms'))
+
+    SiteSetting.create!(
+      setting: Setting.find_by(name: 'ConfirmTerms'),
+      value: false,
+      provider: 'greenlight'
+    )
```

Comment:
You're doing a data migration inside of a schema migration. They should be seperated into 2 separate migrations